internal/server/server.go

```diff
@@ -42,6 +43,7 @@ func NewServer(realm string, a AuthHandler) *Server {
 		authHandler:        a,
 		manager:            &allocation.Manager{},
 		reservationManager: &allocation.ReservationManager{},
+		permissionLifetime: 5 * time.Minute,
```

Comment:
Can we keep the value before `10 minutes`

beyond that LGTM! If you can change that will approve, thanks @songjiayang :)

Reply:
@Sean-Der now move channelBindTimeout to StartArguments, and change it to 10 minutes with default.